app/controllers/erdbeere_controller.rb

```diff
@@ -8,7 +8,7 @@ def current_ability
   end
 
   def show_example
-    response = Faraday.get(ENV.fetch("ERDBEERE_API") + "/examples/#{params[:id]}")
+    response = Clients::ErdbeereClient.get("examples/#{params[:id]}")
```

Comment:
Depending on what we think is the future of Erdbeere, we'd need some tests here as well for the Erdbeere endpoints. However, I see that this is non-trivial since it's communicating with yet another service, so maybe this is something for integratino tests and then test the individual endpoints themselves in Erdbeere. In light of the fact that upgrading the erdbeere PR was rather a rescue effort, this might be done at a later point in time.

Reply:
This is certainly true. I agree that we should do this at a later point in time.